### PR TITLE
Array manipulation 1

### PR DESCRIPTION
**Pull request by Matthew-Abebe**
Screenshot (319)
Screenshot (320)

### CODE REVIEW

File: array-manipulation-1/src/reverse.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* exported reverse */
+
+function reverse(array) {
+  var newArr = [];
+
+  for (var i = array.length - 1; i >= 0; i--) {
+    newArr.push(array[i]);
+  }
+  return (newArr);
```
Comment (codingcodymiller) [inline array-manipulation-1/src/reverse.js:9]: You don't need to be putting parentheses around your returns like this. You should remove them.

File: array-manipulation-1/src/tail.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/* exported tail */
+
+function tail(array) {
+  var newArr = [];
+
+  for (var i = 1; i < array.length; i++) {
+    var item = array[i];
+    newArr.push(item);
+  }
+  return (newArr);
```
Comment (codingcodymiller) [inline array-manipulation-1/src/tail.js:10]: You don't need to be putting parentheses around your returns like this. You should remove them.

File: array-manipulation-1/src/last.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+/* exported last */
+
+function last(array) {
+  return (array[array.length - 1]);
```
Comment (codingcodymiller) [inline array-manipulation-1/src/last.js:4]: You don't need to be putting parentheses around your returns like this. You should remove them.

File: array-manipulation-1/src/initial.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+/* exported initial */
+
+function initial(array) {
+  var newArr = [];
+
+  for (var i = 0; i < array.length - 1; i++) {
+    var item = array[i];
+    newArr.push(item);
+  }
+  return (newArr);
```
Comment (codingcodymiller) [inline array-manipulation-1/src/initial.js:10]: You don't need to be putting parentheses around your returns like this. You should remove them.

File: array-manipulation-1/src/compact.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+function compact(array) {
+  var newArr = [];
+  for (var i = 0; i < array.length; i++) {
+    if (array[i] !== false && array[i] !== 0 && typeof array[i] !== 'undefined' && array[i] !== '' && array[i] !== null && !Number.isNaN(array[i])
+    ) {
+      newArr.push(array[i]);
+    }
+  }
+  return (newArr);
```
Comment (codingcodymiller) [inline array-manipulation-1/src/compact.js:11]: You don't need to be putting parentheses around your returns like this. You should remove them.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by Matthew-Abebe** [comment]
Screenshot (327)
Screenshot (328)

File: array-manipulation-1/src/compact.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+/* exported compact */
+
+function compact(array) {
+  var newArr = [];
+  for (var i = 0; i < array.length; i++) {
+    if (array[i] !== false && array[i] !== 0 && typeof array[i] !== 'undefined' && array[i] !== '' && array[i] !== null && !Number.isNaN(array[i])
```
Comment (scbowler) [inline array-manipulation-1/src/compact.js:6]: There is probably an easier way to check if something is falsy... 🤔

**Review by scbowler — APPROVED** [review]
Looks good 👍 

One FYI comment below for `compact`, no change required

**Pull request merged**